cloudify_rest_client/secrets.py

```diff
@@ -126,6 +133,17 @@ def get(self, key):
         response = self.api.get('/secrets/{0}'.format(key))
         return Secret(response)
 
+    def export(self, **kwargs):
+        """
+        Returns a list of secrets to be exported
```

Comment:
Do you think it's clear enough? Why not `exports secrets to file`?

Reply:
I think it's a good idea, but won't it be a bit confusing when all the other functions are named the same as in the cli file?